deployment/masternode/ingress.yaml

```diff
@@ -1,15 +1,15 @@
-# apiVersion: extensions/v1beta1
-# kind: Ingress
-# metadata:
-#   name: masternode-ingress
-#   annotations:
-#     kubernetes.io/ingress.class: traefik
-#     traefik.ingress.kubernetes.io/preserve-host: "true"
-#     # traefik.ingress.kubernetes.io/request-modifier: ReplacePathRegex: ^/:7557/(.*) /$1"
-# spec:
-#   rules:
-#   - http:
-#       paths:
-#       - backend:
-#           serviceName: masternode
-#           servicePort: 7557
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRouteTCP
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: traefik
+  name: multichain-ingress-tcp
```

Comment:
`-tcp` kan beter `-web` zijn? Of is port 8571 de TCP port en 8570 de RPC port? Is daar wel een ingress voor?